engines/wintermute/base/base_game.cpp

```diff
@@ -488,6 +491,7 @@ bool BaseGame::initialize1() {
 bool BaseGame::initialize2() { // we know whether we are going to be accelerated
 #ifdef ENABLE_WME3D
 	_renderer = makeOpenGL3DRenderer(this);
+	_renderer3D = static_cast<BaseRenderOpenGL3D *>(_renderer);
```

Comment:
You can even avoid this cast, if you initialize _renderer3D first, then you can simply do
_renderer = _renderer3D;

Afterwards. With the nice benefit that the compiler now gets to check types properly.

Reply:
Actually, at the moment, makeOpenGL3DRenderer returns a BaseRenderer pointer, similar to makeOSystemRenderer. Change this to return the actual type, too?

Reply:
Sure, eventually that should be made into whatever the common super-type for 3d-renderers will be, as that's the only thing anyone should care about outside the internals of the renderer, but since we have a single 3D renderer for now, use that type.

Reply:
post this PR